ui/guest/src/react/hooks.ts

```diff
@@ -61,7 +61,7 @@ interface UseModelProps extends Omit<ICEProps, 'modelId'> {
 
 const handlerMap = {
   mouseover: 'onMouseOver',
-  mouseleave: 'onMouseLeave',
+  mouseout: 'onMouseout',
```

Comment:
Maybe we can keep the react layer with mouseleave and handle both (mouseout & mouseleave) on epics and reducers.